ChromiumHtmlToPdfLib/Protocol/CaptureScreenshotResponse.cs

```diff
@@ -37,7 +37,7 @@ internal class CaptureScreenshotResponse
     #region Properties
     [JsonProperty("id")] public long Id { get; set; }
 
-    [JsonProperty("result")] public CaptureScreenshotResult Result { get; set; }
```

Comment:
For protocols mappings I just marked non-nullable those fields which are used without null checks in code. I don't see nullability information in API documentation which means it should be extracted from somewhere else (e.g. typescript definitions). Doesn't look important enough to bother